dev-haskell/clash-cores/clash-cores-1.8.1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.4.0
+# manually udpdated to use the GitHub repo instead of hackage
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal git-r3
+
+DESCRIPTION="A collection of IP cores for Clash"
+HOMEPAGE="https://github.com/clash-lang/clash-compiler"
+
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/clash-lang/clash-compiler"
+EGIT_COMMIT="v1.8.1"
```

Comment:
```
   VisibleVcsPkg: version 1.8.1: VCS version visible for KEYWORDS="~amd64",
```

You're getting a CI error here because `git-r3` is not allowed except for packages with no keywords. It's better to use a tarball from github instead of git, e.g.

https://github.com/clash-lang/clash-compiler/archive/3f5dc67c0e526e43a4dd88eb3902e39ed512c166.tar.gz

You'll have to do the same for `clash-ffi` and `clash-prelude-hedgehog`.

Reply:
You can see an example of how to do this here:

https://github.com/gentoo-haskell/gentoo-haskell/blob/7f612790664de2a7499b411490f85382ffaa5300/dev-haskell/tmp-postgres/tmp-postgres-1.35.0.0_pre20230808.ebuild#L21-L27

We also prefer that you add a `pre` suffix to the name of the ebuild, which corresponds to the date of the commit you're using (`pre20230808` for that example).

Reply:
Thanks for feedback. I've updated it.